pandas/core/series.py

```diff
@@ -117,11 +117,9 @@ class Series(base.IndexOpsMixin, strings.StringAccessorMixin,
     data : array-like, dict, or scalar value
         Contains data stored in Series
     index : array-like or Index (1d)
-        Values must be unique and hashable, same length as data. Index
-        object (or other iterable of same length as data) Will default to
-        RangeIndex(len(data)) if not provided. If both a dict and index
-        sequence are used, the index will override the keys found in the
-        dict.
+        Values must be hashable and the same length as data. Will default to
+        RangeIndex(len(data)) if not provided. If both a dict and index sequence
```

Comment:
This line is too long (therefore travis is failing, we test for PEP8 (line length < 80))